### ISSUE TITLE
Is not reactive..?

### ISSUE THREAD

**Issue by eminx** [adtribute/pub-sub-lite#8]
I realise that, using React along with `withTracker` HOC from `react-meteor-data`, the subscriptions are not reactive. 

For example I had a chat feature that now requires me to reload the page to see the new messages ;]

Is this meant to be?

**Comment by npvn** [comment]
Yes, pub-sub-lite uses Meteor method under the hood to simulate the pub/sub interface. It's most ideally used when you have existing pubs/subs that you want to replace with methods (e.g. for performance purposes) without having to rewrite the whole client and server-side codebase. The obvious trade-off is that there's no more reactivity. So pub-sub-lite should only be used in places where reactivity is not essential. 

For use cases where you don't need full reactivity across clients, but still want to have UI reactively updated after a mutation *for the method caller*, [enhanced methods](https://github.com/adtribute/pub-sub-lite#emitting-mutation-update-messages) may help.
